### PR TITLE
Update guide_azure.rst

### PR DESCRIPTION
**Pull request by sey**
##### SUMMARY
Fix spacing and typos

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

### CODE REVIEW

File: docs/docsite/rst/scenario_guides/guide_azure.rst [hunk 1]
```diff
@@ -267,13 +267,13 @@ consider the following:
 * When creating a VM with the ``azure_rm_virtualmachine`` module,
   you need to explicitly set the ``managed_disk_type`` parameter
   to change the OS disk to a managed disk.
-  Otherwise, the OS disk becomes  an unmanaged disk..
-* When you create a data disk with  the ``azure_rm_manageddisk`` module,
-  you need to  explicitly specify the  ``storage_account_type`` parameter
-  to make it a  managed disk.
+  Otherwise, the OS disk becomes  an unmanaged disk.
```
Comment (Akasurde) [inline docs/docsite/rst/scenario_guides/guide_azure.rst]:
```suggestion
  Otherwise, the OS disk becomes an unmanaged disk.
```

**Comment by acozine** [comment]
This is an improvement, thanks @sey and @Akasurde. I've committed the last whitespace correction and will merge as soon as the latest commit passes our CI suite.

**Comment by acozine** [comment]
rebuild_merge /azp run

**Comment by Akasurde** [comment]
rebuild_merge

**Pull request merged**